src/Tests/Microsoft.Diagnostics.Monitoring.TestCommon/TemporaryDirectory.cs

```diff
@@ -17,7 +17,7 @@ public TemporaryDirectory(ITestOutputHelper outputhelper)
         {
             _outputHelper = outputhelper;
 
-            _directoryInfo = Directory.CreateDirectory(Path.Combine(Path.GetTempPath(), Guid.NewGuid().ToString("N")));
+            _directoryInfo = Directory.CreateDirectory(Path.Combine(Path.GetTempPath(), Path.GetRandomFileName()));
```

Comment:
Are we deleting this folder afterwards? Note that GetRandomFileName likely shares some of the limitations of GetTempFile such as a limit of 65k entries.

Reply:
Yes, it is deleted on dispose